lib/encryption/aes.ex

```diff
@@ -63,6 +63,7 @@ defmodule Encryption.AES do
   # as above but *asumes* `default` (latest) encryption key is used.
   @spec decrypt(any) :: String.t
   def decrypt(ciphertext) do
+    IO.inspect ciphertext, label: "ciphertext"
```

Comment:
Do you want to keep this IO.inspect to see what is passed to the function?

Reply:
Ah, good catch. I added that in to debug your question ... https://github.com/dwyl/app/issues/250#issuecomment-559568401
removing it now. 👍